addons/block_code/drag_manager/drag_manager.gd

```diff
@@ -39,38 +41,41 @@ func _process(_delta):
 		var closest_snap_point: SnapPoint = null
 		var closest_dist: float = INF
 		var snap_points: Array[Node] = get_tree().get_nodes_in_group("snap_point")
-		for n in snap_points:
-			if n is SnapPoint:
-				var snap_point: SnapPoint = n as SnapPoint
-				if snap_point.block == null:
-					push_error("Warning: a snap point does not reference it's parent block.")
-					continue
-				if snap_point.block.on_canvas:
-					if Types.can_cast(dragging.block_type, snap_point.block_type):
-						var snap_global_pos: Vector2 = snap_point.get_global_rect().position
-						var temp_dist: float = dragging_global_pos.distance_to(snap_global_pos)
-						if temp_dist < closest_dist:
-							# Check if any parent node is this node
-							var is_child: bool = false
-							var parent = snap_point
-							while parent is SnapPoint:
-								if parent.block == dragging:
-									is_child = true
-
-								parent = parent.block.get_parent()
-
-							if not is_child:
-								closest_dist = temp_dist
-								closest_snap_point = snap_point
-
-		if closest_dist > 80.0:
-			closest_snap_point = null
+		for snap_point in snap_points:
+			if not snap_point is SnapPoint:
+				push_error('Warning: a node in group "snap_point"snap is not of class SnapPoint.')
```

Comment:
I don't think you mean to include the literal `"snap_point"snap` string in the message here?